tools/d2g/d2g.go

```diff
@@ -98,7 +98,8 @@ func ConvertTaskDefinition(dwTaskDef json.RawMessage) (json.RawMessage, error) {
 // a converted Docker Worker task payload (see d2g.Convert function) to run
 // Docker Worker tasks under Generic Worker.
 func Scopes(dwScopes []string, dwPayload *dockerworker.DockerWorkerPayload, taskQueueID string) (gwScopes []string) {
-	gwScopes = []string{}
+	// scopes to use docker, by default, should just come "for free"
+	gwScopes = []string{"generic-worker:os-group:" + taskQueueID + "/docker"}
```

Comment:
It occurs to me we could look at dwPayload to see if the engine is set to docker or podman, and only add it if docker is used. Normally we shouldn't add scopes based on what the payload is, but I think in this case it might help restrict the scopes a bit.